### DISCUSSION TITLE
featured/cover/thumb images not showing

### DISCUSSION THREAD

**Discussion by ghost** [jpanther/congo#407] [Q&A]
Trying to add 'featured' mages to my posts.

Using a page bundle technique have tried adding cover.jpg and thumb.jpg into directory of post but not arriving in render.  Have added a test image into post to check that is being pulled through.  Have looked at others sites and can't work out why it isn't working

Example post is: https://mark.hysted.com/back-to-the-commute/

Using cloudlfare via GitHub.

**Comment by gcailly** [comment]
Hi,
I have the same problem. I defined thumnails and a logo too but they do not show.
The host is Hostinger, but it's the same in local.
I use Hugo 0.108.
